client/rest-high-level/src/test/java/org/elasticsearch/client/CCRIT.java

```diff
@@ -148,8 +170,11 @@ public void testAutoFollowing() throws Exception {
         assertThat(response.isAcknowledged(), is(true));
 
         assertBusy(() -> {
-            assertThat(indexExists("copy-logs-20200101"), is(true));
+            CcrStatsRequest ccrStatsRequest = new CcrStatsRequest();
+            CcrStatsResponse ccrStatsResponse = execute(ccrStatsRequest, ccrClient::getCcrStats, ccrClient::getCcrStatsAsync);
+            assertThat(ccrStatsResponse.getAutoFollowStats().getNumberOfSuccessfulFollowIndices(), equalTo(1L));
         });
+        assertThat(indexExists("copy-logs-20200101"), is(true));
```

Comment:
how come you pulled this out of the assertBusy? just making sure it wasnt an omission.

Reply:
If the `numberOfSuccessfulFollowIndices` is 1 then it means that the follower index should already have been created. So there is no need to place the check whether `copy-logs-20200101` exists in the assertBusy. In fact if it isn't there immediately after `numberOfSuccessfulFollowIndices` reports 1 then it is a bug.